### PR TITLE
Shared image gallery

### PR DESCRIPTION
**Pull request by cesar-younes**
Add samples for Storing in and Reading from Azure Shared Image Gallery

### CODE REVIEW

File: image-template/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Custom image as Packer base
+This template demonstrates use a custom image as the starting point for the Packer template
```
Comment (jcorioland) [inline image-template/README.md]: Do you think it's worth to add some extra details about what are the different properties inside the `image-template.json` file? Might be useful to describe in the readme the different input variables and highlight the parts of the json that are relevant for a custom image? Like you did for the shared image gallery.

**Review by jcorioland — APPROVED** [review]
Not a lot to add to the files that are part of the PR, it is clear. I've just let a comment for the image-template/README.md file that is a bit light. Otherwise, looks good to me. I will read the other parts that are already on master to make sure it's consistent and will make some comments if needed.

File: image-template/README.md (hunk 1 above)
Comment (jcorioland) [reply image-template/README.md]: You can actually ignore my previous comment, everything is well documented into the main README :)

Comment (cesar-younes) [reply image-template/README.md]: I just forgot about this one. It's still light though because there's not much to say.

**Pull request merged**